roles/smartgateway/templates/ceilometer-metrics-configmap.yaml.j2

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
To clarify, this file is called `metrics-configmap.yaml.j2` on master and was previously removed from this branch, but is still needed for ceilometer metrics with the legacy SG.